### PR TITLE
add schema examples

### PR DESCRIPTION
**Pull request by fanglifl**

### CODE REVIEW

File: generate_schema/1D_Z2.jl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+U1 = -2π * 10;
+U2 = 2π * 10;
+Δ = piecewise_linear(clocks = [0.0, 0.6, 2.1, total_time], values = [U1, U1, U2, U2]);
+
+nsites = 9
+atoms = generate_sites(ChainLattice(), nsites, scale = 5.72)
+
+
+H = rydberg_h(atoms; Δ, Ω)
+h = BloqadeSchema.to_json(H,waveform_tolerance=1e-1)
```
Comment (Roger-luo) [inline generate_schema/1D_Z2.jl:21]: maybe we should just export `to_json` function from `BloqadeSchema`

**Comment by Wang-Shengtao** [comment]
Would it be better to put this folder inside BloqadeSchema?

**Comment by fanglifl** [comment]
> can we move this to the `BloqadeSchema/examples` as @Wang-Shengtao commented?

Yes. I will open a new branch after @weinbe58 merged his Rounding schema values PR. I will put these examples under the BloqadeSchema/examples.

**Comment by weinbe58** [comment]
After #412, we should be able to handle all examples @fanglifl has created.

**Comment by weinbe58** [comment]
@fanglifl can you remove the `BloqadeSchema.` from the `BloqadeSchema.to_json`?

**Comment by fanglifl** [comment]
> @fanglifl can you remove the `BloqadeSchema.` from the `BloqadeSchema.to_json`?

yes, I will fix that

**Comment by fanglifl** [comment]
@weinbe58 right now, the only example that throws errors is the local_detuning/local2.jl
It seems to give the same error as I had before

**Review by weinbe58 — APPROVED** [review]

**Pull request merged**